docs/Meadow/Meadow.OS/Networking/index.md

```diff
@@ -77,6 +82,31 @@ async Task ScanForAccessPoints(IWiFiNetworkAdapter wifi)
 }
 ```
 
+# Ethernet
+
+If you're using an [Dual, Switching Ethernet Add-on module](https://store.wildernesslabs.co/collections/meadow-core-compute/products/dual-ethernet-add-on), or a [Meadow F7v2 Core-Compute Dev Kit](https://store.wildernesslabs.co/collections/frontpage/products/meadow-f7v2-core-compute-breakout-board), you can easily use Ethernet network connectivity by switching the `DefaultInterface` setting in the meadow.config.yaml file:
+
+```yml
+...
+# Network configuration.
+Network:
+  Interfaces:
+    - Name: Ethernet
+      UseDHCP: false
+      IPAddress: 192.168.1.60
+      NetMask: 255.255.255.0
+      Gateway: 192.168.1.254
+#    - Name: Ethernet
+#      UseDHCP: true
+#      IPAddress:
```

Comment:
If you are using DHCP then you do not need the IPAddress, NetMask and Gateway fields at all.  Blank one are OK but they can be removed if you like.